README.md

```diff
@@ -32,13 +32,29 @@ pip install .[all]
 ```
 
 ## Using conda
+
+If you are using conda, you can install either the dev or the tag TimeChart build:
+
+```sh
+conda install timechart -c tidair-dev -c pydm-tag
+```
+
+or
+
+```sh
+conda install timechart -c tidair-tag -c pydm-tag
+```
+
+`tidair-dev` is to be built with the latest code, and `tidair-tag` is to be built with the latest release (tag).
```

Comment:
It is better to say something like this:

‘tidair-dev’ has the latest development code, and ‘tidair-tag’ has the latest stable released version (tag).